standalonecontroller/image.yaml

```diff
@@ -0,0 +1,304 @@
+      description: A list of comma-separated directories used for installation and configuration of artifacts for the image during the S2I process.
+      example: "custom,shared"
+    - name: "CONTAINER_HEAP_PERCENT"
+      example: 0.5
+      description: Deprecated.  See JAVA_MAX_MEM_RATIO.
+    - name: "INITIAL_HEAP_PERCENT"
+      example: 0.5
+      description: Deprecated.  See JAVA_INITIAL_MEM_RATIO.
+    - name: "PROBE_DISABLE_BOOT_ERRORS_CHECK"
+      example: "true"
+      description: Disable the boot errors check in the probes.
+ports:
+    - value: 8443
+    - value: 8778
+cmd:
+    - "/opt/eap/bin/openshift-launch.sh"
+packages:
+    - rh-mongodb32-mongo-java-driver
+    - postgresql-jdbc
+    - mysql-connector-java
+    - rh-maven33
+    - hostname
+    - python-requests
+    - python-enum34
+    - PyYAML
+sources:
+    - url: https://maven.repository.redhat.com/ga/org/jolokia/jolokia-jvm/1.3.6.redhat-1/jolokia-jvm-1.3.6.redhat-1-agent.jar
+      md5sum: 75e5b5ba0b804cd9def9f20a70af649f
+    - url: javax.json-1.0.4.jar
+      md5sum: 569870f975deeeb6691fcb9bc02a9555
+    - url: jboss-logmanager-ext-1.0.0.Alpha2-redhat-1.jar
+      md5sum: 7c743e35463db5f55f415dd666d705c5
+    - artifact: openshift-ping-common-1.1.5.Final-redhat-1.jar
+      md5: e27cdcf626ab1366c8d7d18ebd399ec4
+    - artifact: openshift-ping-dns-1.1.5.Final-redhat-1.jar
+      md5: ecab0d5ad969cb001e778305485b4640
+    - artifact: openshift-ping-kube-1.1.5.Final-redhat-1.jar
+      md5: 2f74acc3efc68e3781b7004f5a683fdc
+    - url: oauth-20100527.jar
+      md5sum: 91c7c70579f95b7ddee95b2143a49b41
+    - url: activemq-rar-5.11.0.redhat-621084.rar
+      md5sum: 207e17ac8102c93233fe2764d1fe8499
+    - url: hawkular-javaagent-1.0.0.CR5-redhat-1-shaded.jar
+      md5sum: f5d3e0220e10c706ef86af84771cf74e
+    - url: rh-sso-7.0.0-eap7-adapter.zip
+      md5sum: 1542c1014d9ebc24522839a5fa8bee4d
+    - url: rh-sso-7.0.0-saml-eap7-adapter.zip
+      md5sum: ce858a47c707b362a968ffd5c66768dd
+cct:
+    - name: BxMS
+      modules:
+          - url: https://github.com/jboss-openshift/cct_module
+      changes:
+          - cct_module.dynamic-resources:
+                - install_sh:
+          - cct_module.s2i-common:
+                - install_sh:
+          - cct_module.java-alternatives:
+                - run_sh:
+          - cct_module.os-eap7-openshift:
+                - prepare_sh:
+          - cct_module.os-eap-s2i:
+                - prepare_sh:
+          - cct_module.os-java-jolokia:
+                - install_as_root:
+          - cct_module.jolokia:
+                - user: 185
+                - configure_sh:
+          - cct_module.os-eap7-openshift:
+                - user: 185
+                - configure_sh:
+          - cct_module.os-eap7-modules:
+                - user: 185
+                - configure_sh:
+          - cct_module.os-eap7-ping:
+                - user: 185
+                - configure_sh:
+          - cct_module.os-eap70-openshift:
```

Comment:
Without this module the container didn't start properly (file `standalone-openshift.xml` not found). I think the `bpmsuite70-executionserver-openshift` has this problem. Wdyt @goldmann ?